src/InterventionBackend.php

```diff
@@ -30,6 +35,36 @@ class InterventionBackend implements Image_Backend, Flushable
     const CACHE_DIMENSIONS = 'DIMENSIONS_';
 
     /**
+     * How long to cache each error type
+     *
+     * @config
+     * @var array Map of error type to config.
+     * each config could be a single int (fixed cache time)
+     * or list of integers (increasing scale)
+     */
+    private static $error_cache_ttl = [
+        self::FAILED_INVALID => 0, // Invalid file type should probably never be retried
+        self::FAILED_MISSING => '5,10,20,40,80', // Missing files may be eventually available
```

Comment:
Yeah, it's too hard to override a non-associative array.

You can still override the string with an array, or vice versa, however. :)